src/plugins/data/public/ui/filter_bar/filter_editor/index.tsx

```diff
@@ -245,7 +245,7 @@ class FilterEditorUI extends Component<Props, State> {
   private renderRegularEditor() {
     return (
       <div>
-        <EuiFlexGroup responsive={false} gutterSize="s">
+        <EuiFlexGroup responsive={true} gutterSize="s">
```

Comment:
Does this make it break to separate lines too soon?

Reply:
It makes it break ~`250px` sooner than ideal. 
![Screenshot 2020-10-05 at 18 42 48](https://user-images.githubusercontent.com/7784120/95107732-9852e880-073a-11eb-8e7c-3051b43d31d3.png)


Would you prefer to make it break with custom SASS instead? or is there a way to make it nicely using EUI api?

Reply:
No EUI doesn't have a good way of changing the responsive breakpoint yet without completely custom SASS. If your screenshot is of where it starts to break, I think it's fine.